docs/mlcroissant/Field.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="https://cdn.simplecss.org/simple.min.css">
+<html>
+<head>
+    <title>Field - Croissant &#129360</title>
+</head>
+<header>
+    <nav>
+        <a href="./Metadata.html">Metadata</a>
+        <a href="./FileObject.html">FileObject</a>
+        <a href="./FileSet.html">FileSet</a>
+        <a href="./RecordSet.html">RecordSet</a>
+        <a href="./Field.html">Field</a>
+    </nav>
+    <h1>Croissant &#129360</h1>
+    Croissant specifications as implemented by <code><a href="https://pypi.org/project/mlcroissant">mlcroissant</a></code>.
+    <br>
+    For the actual specifications, please refer to the <a href="https://mlcommons.org/croissant/1.0">Croissant 1.0 standard.</a>.
+</header>
+<body>
+    <h3>Field</h3>
+    <span>Nodes to describe a dataset Field.</span>
```

Comment:
Is this a copy paste from the specs? Or are we adding more content? If the former, than maybe it makes more sense to just add a link to the specs here, so that we don't need to be sure that this docs get updated?

Reply:
Oh, sorry -- this is generated by scripts/documentation.py no?

Reply:
The HTML files mirror the specs **as implemented by mlcroissant**. This allows to easily compare mlcroissant to croissant as I hope it will get deployed to github.io

Reply:
> Oh, sorry -- this is generated by scripts/documentation.py no?

Yes, exactly! I hope it will get deployed to github.io. Let's try it out :)